src/components/position.js

```diff
@@ -1,8 +1,8 @@
-var coordinatesMixin = require('../utils/coordinates').componentMixin;
 var registerComponent = require('../core/component').registerComponent;
-var utils = require('../utils/');
 
-module.exports.Component = registerComponent('position', utils.extend({
+module.exports.Component = registerComponent('position', {
+  type: 'coordinate',
```

Comment:
Yeah, I thought about it, but we'd have to restructure the schema to be able to define meta values.

```js
{
  schema: {
    type: 'coordinate',
    properties: {
      x: { default: 0 }
     }
   }
}
```